operations/customers.md

```diff
@@ -0,0 +1,169 @@
+        "address2": "Address",
+        "city": "City",
+        "state": "State",
+        "postalCode": "10000",
+        "country": "HR",
+        "notes": "This is a note",
+        "countrySpecificCode": "code",
+        "companyName": "Company name",
+        "createdAt": "2024-10-16T14:30:00Z",
+        "updatedAt": "2024-10-19T14:30:00Z"
+      }
+    }
+  ],
+  "links": {
+    "prev": "https://pos.mews-demo.com/api/v2/customers?page%5Bbefore%5D=NA&page%5Bsize%5D=1",
+    "next": "https://pos.mews-demo.com/api/v2/customers?page%5Bafter%5D=NA&page%5Bsize%5D=1"
+  }
+}
+```
+Below is a list of all possible fields this endpoint can return including relationships fields fetched with include query parameter.
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `fullName` | string,null | optional, max length 255 characters | The full name of the customer. This can be either a personal name or a company name. |
+| `companyName` | string,null | optional, max length 255 characters | The name of the company associated with the customer. |
+| `taxNumber` | string,null | optional, max length 40 characters | The customer's tax identification number. |
+| `email` | string,null | optional, max length 320 characters | The customer's email address. |
+| `address1` | string,null | optional, max length 255 characters | The first line of the customer's address. |
+| `address2` | string,null | optional, max length 255 characters | The second line of the customer's address. |
+| `city` | string,null | optional, max length 100 characters | The city of the customer's address. |
+| `state` | string,null | optional, max length 100 characters | The state or province of the customer's address. |
+| `postalCode` | string,null | optional, max length 20 characters | The postal or ZIP code of the customer's address. |
+| `country` | string,null | optional, max length 100 characters | The country of the customer's address. |
+| `notes` | string,null | optional, max length 500 characters | Additional notes about the customer. |
+| `phone` | string,null | optional, max length 20 characters | The customer's phone number. |
+| `mobile` | string,null | optional, max length 20 characters | The customer's mobile phone number. |
+| `countrySpecificCode` | string,null | optional, max length 50 characters | A unique country specific code assigned to the customer. Lottery code for example in Italy. |
+| `dateOfBirth` | string,null | optional, max length 10 characters | The customer's date of birth in YYYY-MM-DD format. |
+
+#### customer_pagination_links
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `prev` | string | required, max length 1024 characters | The link to the previous page of results. |
+| `next` | string | required, max length 1024 characters | The link to the next page of results. |
+
+## Create customer
+
+A customer represents a person or organization that can be billed for goods or services.
```

Comment:
The description should describe the function, so I would suggest starting with "Create a customer". That may be sufficient in this case.